style.css

```diff
@@ -344,3 +344,167 @@ header {
   height: 0.5rem;
   align-self: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .head-section {
+    flex-direction: column;
+    background-image: url("desktop_header_bg.png");
+    justify-content: center;
+    gap: 1.5rem;
+    padding: 18rem 26rem 26rem 26rem;
+    border-bottom-left-radius: 7rem;
+  }
+
+  header {
+    margin-top: 0;
+    padding: 0 15.5rem;
+  }
+
+  .normal-button {
+    display: none;
+  }
+
+  .nav-ul {
+    display: flex;
+    gap: 3.2rem;
+  }
+
+  .nav-ul li {
+    display: flex;
+  }
+
+  .nav-ul li a {
+    list-style: none;
+    text-decoration: none;
+    padding: 1rem;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .card {
+    flex-direction: row;
+    padding: 2.4rem;
+    margin: 0 1.4rem;
+  }
+
+  .card > * {
+    flex: 1;
+  }
+
+  .card-div {
+    display: flex;
+    flex-direction: column;
+    margin-right: 0;
+  }
+
+  .card4 {
+    flex-direction: row-reverse;
+    order: 3;
+  }
+
+  .card3 {
+    order: 2;
+  }
+
+  .card1 {
+    order: 4;
+  }
+
+  .card2 {
+    order: 5;
+    flex-direction: row-reverse;
+  }
+
+  .card3-img-2 {
+    display: initial;
+  }
+
+  .card3-img-1 {
+    display: none;
+  }
+
+  .about-mySelf {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 16.6rem 14.4rem;
+    gap: 5rem;
+    order: 6;
+  }
+
+  .about-myself-first-section {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 45%;
+  }
+
+  .about-myself-second-section {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    width: 55%;
+  }
+
+  .about-mySelf-div {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .about-mySelf h1 {
+    margin-top: 0;
+  }
+
+  .contact-me {
+    order: 7;
+  }
+
+  form {
+    width: 45rem;
+    margin: 0 auto;
+  }
```

Comment:
- [ ] What do you think about styling the `::placeholder` selector in the `textarea` field and introducing a little padding so it looks similar to the Figma design? Right now the placeholder is attached to the border and it doesn't look quite right visually. Please do implement this feature.

![image](https://user-images.githubusercontent.com/90556221/178961955-136f7172-46ec-4086-8e5a-a5ba1fd338c6.png)